### PR TITLE
Responsive cleanup, featured researcher, recently added, photo creds...

### PR DESCRIPTION
**Pull request by mtribone**
,tweaks,, refs #9391  All kinds of responsive tweaks for background image, marketing text, explore, featured researcher, recent additions layout, etc.

### CODE REVIEW

**Comment by mtribone** [comment]
Restarting build. #708 passed. #709 went sleepies.

File: app/views/homepage/_home_header.html.erb [hunk 1]
```diff
@@ -6,4 +6,7 @@
     <%= link_to "<span class=\"glyphicon glyphicon-upload\"></span> #{t('sufia.share_button')}".html_safe, sufia.new_generic_file_path, class: "btn btn-primary btn-lg btn-block", id: "contribute_link" %>
     <p class="text-center"><a href="/terms/">Terms of Use</a></p>
   </div>
+  <div class="photo_credit">
+    <a href="#">Photo and image credits.</a>
```
Comment (carolyncole) [inline app/views/homepage/_home_header.html.erb:10]: I thought this was going to point at version/photo_credit.  We can change it after this gets merged though...

**Comment by carolyncole** [comment]
I have restarted travis.  Once that passes I am good to merge.  We can clean up any little issues after we look at it on the servers.

File: app/views/homepage/_home_header.html.erb (hunk 1 above)
Comment (mtribone) [inline app/views/homepage/_home_header.html.erb:10]: Oops. I forgot. Sorry! Yes a quick change after merge.

**Pull request merged**